### ISSUE TITLE
Module Mapping Regression (conflicting mapping at ...)

### ISSUE THREAD

**Issue by joeljeske** [bazelbuild/rules_nodejs#1595]
# 🐞 bug report

### Affected Rule

 The issue is caused by the rule: nodejs_binary

### Is this a regression? 

 
 Yes, the previous version in which this bug was not present was: 1.1.0

### Description

I believe the bug is incorrect boolean logic when deciding to use a module mapping or not. 

I believe the issue is in this section here: 

https://github.com/bazelbuild/rules_nodejs/commit/110e00e#diff-3238816daaca23ffcb6861ceb46dfc2aR44-R47

Perhaps the change needed is this?

```diff
           if mappings[k][0] == "runfiles":
                return True
-         elif v[0] != "runfiles":
+         elif v[0] == "runfiles":
               return False
``` 

## 🔬 Minimal Reproduction

I'm not entirely sure how to reproduce this, as it occurs in a setup with a lot of surrounding rules, however I suspect that @gregmagolan might be able to help me with reproduction as he seemed to be working on this issue very recently. 

## 🔥 Exception or Error

<pre><code>
conflicting mapping at //some:target: some/module-name maps to both ["bin", "ws/some/module-name"] and ["runfiles", "ws/some/module-name"]
</code></pre>

## 🌍  Your Environment

**Operating System:**

<pre>
  <code>
macOS 10.14.6
  </code>
</pre>

**Output of `bazel version`:**

<pre>
  <code>
Build label: 1.2.0
Build target: bazel-out/darwin-opt/bin/src/main/java/com/google/devtools/build/lib/bazel/BazelServer_deploy.jar
Build time: Wed Nov 20 15:08:12 2019 (1574262492)
Build timestamp: 1574262492
Build timestamp as int: 1574262492
  </code>
</pre>

**Rules_nodejs version:**

(Please check that you have matching versions between WORKSPACE file and `@bazel/*` npm packages.)

<pre>
  <code>
v1.2.2
  </code>
</pre>

**Anything else relevant?**

**Comment by gregmagolan** [comment]
Thanks @joeljeske . You are correct that the boolean is backwards and the integration test didn't exercise that path. #1597 will fix it and I'll cut another patch release.

**Comment by joeljeske** [comment]
Awesome thanks for quick feedback!

**Comment by gregmagolan** [comment]
Np. Pushed a quick 1.2.4 release that includes the fix since it fixes the 1.2.0 regression. https://github.com/bazelbuild/rules_nodejs/releases/tag/1.2.4
